net/respondd/Makefile

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2023 David Bauer <mail@david-bauer.net>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=respondd
+PKG_SOURCE_DATE:=2019-05-01
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/freifunk-gluon/respondd.git
+PKG_SOURCE_VERSION:=58405297e76f97d4752dc7511dc15bbc7a01e586
+PKG_MIRROR_HASH:=22bc00df280fe611cfc895528f17269f03af804ea4c2ca2dc62da2457d3a1ee0
+
+PKG_MAINTAINER:=David Bauer <mail@david-bauer.net>
+PKG_LICENSE:=BSD-2-Clause
```

Comment:
Hmm, I dont see any LICENSE in that repository.

Reply:
See the source-file. As well as the package Makefile contained in the Repo.

Reply:
I've checked it, but none files in src includes license header, there is no separate license file and only where there is something about BSD-2-Clause is here https://github.com/freifunk-gluon/respondd/blob/58405297e76f97d4752dc7511dc15bbc7a01e586/Makefile#L6

Reply:
How about https://github.com/freifunk-gluon/respondd/blob/master/src/respondd.c#L7 ?